sourcegraph/about
# Incident response use case /non-blocking feedback

This closes #5287. Here, we implement the remaining feedback from our Marketing stakeholders, re: copy updates, a changed icon, and logo treatment.

The QA doc can be found [here](https://docs.google.com/document/d/18dNP6QihejP8CVSm-tdmbH20D5beFYlGXyBme0286U4/edit#).

### Notes
- Crosshairs icon changed to magnifying glass
- Copy changes in `CustomCarousel`
- Copy changes in blog descriptions
- Reformat "What does it mean for you?" on this and other Use Case pages
- Center and slightly enlarge Cloudflare logo in feature box

### Testing
- Open `use-cases/incident-response`
- Check that the updates marked "Non-Blocking" in the QA doc are accurate
- Test the static build
